test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,27 @@
 const { fizzBuzz } = require("../src");
```

Comment:
All tests are passing, very nice :), and well done for using ""

It might be worth to start to think about edge cases, for example what would happen if you pass in a negative number to the function? or a very large number (bigInt), would your function handle these cases or would it break?
These are not included in this exercise and you do not have to add them, I am only mentioning it just for you to think about, how can you make your function more secure by testing for edge cases.